rust/docker-init.sh

```diff
@@ -1,32 +1,32 @@
 #!/bin/sh
 
 if [ "${FIREZONE_ENABLE_MASQUERADE}" = "1" ]; then
-  IFACE="tun-firezone"
-  # Enable masquerading for ethernet and wireless interfaces
-  iptables -C FORWARD -i $IFACE -j ACCEPT > /dev/null 2>&1 || iptables -A FORWARD -i $IFACE -j ACCEPT
-  iptables -C FORWARD -o $IFACE -j ACCEPT > /dev/null 2>&1 || iptables -A FORWARD -o $IFACE -j ACCEPT
-  iptables -t nat -C POSTROUTING -o e+ -j MASQUERADE > /dev/null 2>&1 || iptables -t nat -A POSTROUTING -o e+ -j MASQUERADE
-  iptables -t nat -C POSTROUTING -o w+ -j MASQUERADE > /dev/null 2>&1 || iptables -t nat -A POSTROUTING -o w+ -j MASQUERADE
-  ip6tables -C FORWARD -i $IFACE -j ACCEPT > /dev/null 2>&1 || ip6tables -A FORWARD -i $IFACE -j ACCEPT
-  ip6tables -C FORWARD -o $IFACE -j ACCEPT > /dev/null 2>&1 || ip6tables -A FORWARD -o $IFACE -j ACCEPT
-  ip6tables -t nat -C POSTROUTING -o e+ -j MASQUERADE > /dev/null 2>&1 || ip6tables -t nat -A POSTROUTING -o e+ -j MASQUERADE
-  ip6tables -t nat -C POSTROUTING -o w+ -j MASQUERADE > /dev/null 2>&1 || ip6tables -t nat -A POSTROUTING -o w+ -j MASQUERADE
+    IFACE="tun-firezone"
+    # Enable masquerading for ethernet and wireless interfaces
+    iptables -C FORWARD -i $IFACE -j ACCEPT >/dev/null 2>&1 || iptables -A FORWARD -i $IFACE -j ACCEPT
+    iptables -C FORWARD -o $IFACE -j ACCEPT >/dev/null 2>&1 || iptables -A FORWARD -o $IFACE -j ACCEPT
+    iptables -t nat -C POSTROUTING -o e+ -j MASQUERADE >/dev/null 2>&1 || iptables -t nat -A POSTROUTING -o e+ -j MASQUERADE
+    iptables -t nat -C POSTROUTING -o w+ -j MASQUERADE >/dev/null 2>&1 || iptables -t nat -A POSTROUTING -o w+ -j MASQUERADE
+    ip6tables -C FORWARD -i $IFACE -j ACCEPT >/dev/null 2>&1 || ip6tables -A FORWARD -i $IFACE -j ACCEPT
+    ip6tables -C FORWARD -o $IFACE -j ACCEPT >/dev/null 2>&1 || ip6tables -A FORWARD -o $IFACE -j ACCEPT
+    ip6tables -t nat -C POSTROUTING -o e+ -j MASQUERADE >/dev/null 2>&1 || ip6tables -t nat -A POSTROUTING -o e+ -j MASQUERADE
+    ip6tables -t nat -C POSTROUTING -o w+ -j MASQUERADE >/dev/null 2>&1 || ip6tables -t nat -A POSTROUTING -o w+ -j MASQUERADE
 fi
 
 if [ "${LISTEN_ADDRESS_DISCOVERY_METHOD}" = "gce_metadata" ]; then
-  echo "Using GCE metadata to discover listen address"
+    echo "Using GCE metadata to discover listen address"
 
-  if [ "${PUBLIC_IP4_ADDR}" = "" ]; then
-    public_ip4=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google" -s)
-    export PUBLIC_IP4_ADDR="${public_ip4}"
-    echo "Discovered PUBLIC_IP4_ADDR: ${PUBLIC_IP4_ADDR}"
-  fi
+    if [ "${PUBLIC_IP4_ADDR}" = "" ]; then
+        public_ip4=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google" -s)
+        export PUBLIC_IP4_ADDR="${public_ip4}"
```

Comment:
@jamilbk do we use 4 spaces tab spacing for shell scripts? Elixir and Rust are using 2 spaces so it will lead to constant conflicts (editors will do one thing or another and CI will fail forcing people to go back and change spacing).

Reply:
@AndrewDryga, I know in Vim you can set your tab preferences on a per file type basis.  Could the same be done for VSCode? (I'm guessing yes, but I'm a VSCode n00b)

Reply:
Rust is using 2 spaces? I thought Rust was using 4 spaces.

Reply:
Rust is 4 spaces -- I used 4 for shell scripts since that's the convention that they all were (except the ones in the swift/ dir).

We can set to whatever. By default, `shfmt` uses 2 tabs I believe.

Reply:
Ah, did not know rust uses tabs. Let's use whatever (VSCode will handle that for me).